actions/send_websocket_MOD.js

```diff
@@ -0,0 +1,84 @@
+module.exports = {
+  name: 'Send Websocket',
+  section: 'Other Stuff',
+  meta: {
+    version: '2.1.7',
+    preciseCheck: false,
+    author: 'DBM Mods',
+    authorUrl: null,
+    downloadURL: null,
+  },
+
+  subtitle() {
+    return `Gets a Transcript from a Channel`;
```

Comment:
Copy and paste mistake?

Reply:
Ohh yeah i forget to change the subtitle

And its ws:// so should i change it to wss:// to make it more secure?

Reply:
> Ohh yeah i forget to change the subtitle
> 
> And its ws:// so should i change it to wss:// to make it more secure?

You should see if you can add an option for it, since ws and was are not compatible with each other

Reply:
Ok i will update it tomorow

Reply:
UPDATE: send websocket should work on wss:// if I add a switch to set to ws:// or wss:// but I can't test it out because I can't host a wss because you need an HTTPS server with an SSL certification